server/routes/events.js

```diff
@@ -4,9 +4,36 @@ const router = express.Router();
 const Event = require("../models/event");
 
 /* GET events listing. */
-router.get("/", function (req, res, next) {
-  const event = {};
-  res.status(200).json({ event });
+//GET all events
+router.get("/", async function (req, res) {
+  try{
+    const event =  await Event.find().exec();
+    if (!event)
+      return res.status(404).json({ error: "No events at this time"});
+    res.status(200).json({ event });
+  } catch (error) {
+    res.status(500).json({ error });
+  }
+});
+
+//GET event by ID
+router.get("/:id", async function (req, res) {
+  try{
+    const { id } = req.params;
+    const event =  await Event.findById(id).exec();
+    if (!event)
+      return res.status(404).json({ error: "Event does not exist"});
+    res.status(200).json({ event });
+  } catch (error) {
+    res.status(400).json({ error });
+  }
+});
+
+//POST event(under the assumption input is correct)
+router.post("/", async function (req, res) {
+  const { event } = req.body;
+  const newEvent = await Event.create(event);
+  res.status(200).json({ newEvent });
 });
```

Comment:
Let's return the object with key `event` vs. with key `newEvent`, so that the response type for this route matches that of GET /event.